test/e2e/util_test.go

```diff
@@ -472,17 +476,31 @@ func verifyExternalIngressController(t *testing.T, name types.NamespacedName, ho
 		}
 	}()
 
+	// If we have a DNS as an external IP address, make sure we can resolve it before moving on.
+	// This just limits the number of "could not resolve host" errors which can be confusing.
+	if net.ParseIP(address) == nil {
+		if err := wait.PollImmediate(10*time.Second, 5*time.Minute, func() (bool, error) {
```

Comment:
That's a long wait in an e2e test, can we get away with 1m?

Reply:
It's not long considering DNS can take very well up to 5 mins to propagate for AWS. Keep in mind, these times are dependent on external processes, not something we have control over. I think we want to give platforms as much time as it needs.